langchain/document_loaders/youtube.py

```diff
@@ -146,13 +146,15 @@ def __init__(
         self,
         video_id: str,
         add_video_info: bool = False,
-        language: str = "en",
+        language: List[str] = ["en"],
```

Comment:
Could we use `Union[str, Sequence[str]]` to avoid making breaking changes for existing users, can keep default as `"en"` rather than list and add a big of logic below

Reply:
thanks for your advice, updated in new [commit](https://github.com/hwchase17/langchain/pull/5692/commits/db4bff91570f106831db7695fec3d7e77ec5359a)